src/NServiceBus.Transport.RabbitMQ.AcceptanceTests/ConfigureEndpointRabbitMQTransport.cs

```diff
@@ -1,34 +1,31 @@
 ﻿using System;
 using System.Data.Common;
-using System.Linq;
 using System.Threading.Tasks;
 using NServiceBus;
 using NServiceBus.AcceptanceTesting.Support;
-using NServiceBus.Configuration.AdvancedExtensibility;
-using NServiceBus.Transport;
+using NServiceBus.Transport.RabbitMQ;
 using RabbitMQ.Client;
 
 class ConfigureEndpointRabbitMQTransport : IConfigureEndpointTestExecution
 {
     DbConnectionStringBuilder connectionStringBuilder;
-    QueueBindings queueBindings;
 
     public Task Configure(string endpointName, EndpointConfiguration configuration, RunSettings settings, PublisherMetadata publisherMetadata)
     {
         var connectionString = Environment.GetEnvironmentVariable("RabbitMQTransport_ConnectionString");
 
         if (string.IsNullOrEmpty(connectionString))
         {
-            throw new Exception("The 'RabbitMQTransport_ConnectionString' environment variable is not set.");
+            connectionString = "host=localhost";
+            //throw new Exception("The 'RabbitMQTransport_ConnectionString' environment variable is not set.");
```

Comment:
Should we bring this back?

Reply:
Yep, I brought it back in the latest commit.